src/tests/eet/eet_test_cache.c

```diff
@@ -94,11 +94,12 @@ EFL_START_TEST(eet_test_cache_concurrency)
    thread_ret = eina_thread_join(thread);
    fail_unless(thread_ret == NULL, (char const *)thread_ret);
 
-   eet_close(ef);
+   eet_clearcache();
```

Comment:
```suggestion
   eet_clearcache();
   eet_close(ef);
```
We still need to close that file, otherwise:
```
Running suite(s): Eet
ERR:eet ../src/lib/eet/eet_lib.c:636 eet_shutdown() File 'C:\Users\coku\AppData\Local\Temp\eet_suite_testzWBdba' is still open 1 times !
```

Reply:
And this way it passes the test?

Reply:
> And this way it passes the test?

In both ways it passes, but without closing the file it throws that error.

Reply:
sure, because it needs an `eet_close` to every `eet_open` so `eet` sees it as even.

Reply:
Done.